cookiecutter.json

```diff
@@ -2,7 +2,7 @@
     "pipeline_name": "NF-Something",
     "pipeline_slug": "{{ cookiecutter.pipeline_name.lower().replace(' ', '-') }}",
```

Comment:
If we have `nf-core` as part of the name, we can add the `.replace('/', '-')` code that I suggested in the Dockerfile below. Keeps everything cleaner. eg:

```
"pipeline_slug": "{{ cookiecutter.pipeline_name.lower().replace(' ', '-').replace('/', '-') }}",
```

Reply:
Simpler still: maybe move this to after the `dockerhub_slug` line and then just have:

```
"pipeline_slug": "{{ cookiecutter.dockerhub_slug.replace('/', '-') }}",
```

(saves doing this twice)